detectable/src/main/java/com/synopsys/integration/detectable/detectables/npm/packagejson/CombinedPackageJsonExtractor.java

```diff
@@ -59,7 +59,7 @@ public CombinedPackageJson constructCombinedPackageJson(String rootJsonPath, Str
             
             for(String convertedWorkspace : convertedWorkspaces) {
                 Path workspaceJsonPath =
-                        Path.of(convertedWorkspace + "/package.json").normalize();
+                        Paths.get(convertedWorkspace + "/package.json").normalize();
```

Comment:
As I looked at docs for this method I noticed that it takes multiple arguments which then get joined.

A minor improvement could be to do `Paths.get(convertedWorkspace, "package.json")` instead.

This actually makes me wonder whether the forward slash is a problem in Windows.

Having said all this, I am going to upvote the PR as it's an improvement as is.

Reply:
Thanks for the thoughts. I wonder if I'd have to include the path separator somewhere in there if I went with the multi-argument approach. I'm leaning towards leaving it the way it is give it does seem to test well on both linux based and windows platforms. The thought about Windows was a good one, I ran some tests and found this area in general failed on at least some windows shells. I made a small update if you can take another look.